ws-nextjs-app/pages/[service]/downloads/dataFetch.ts

```diff
@@ -0,0 +1,23 @@
+import 'isomorphic-fetch';
+import { OK } from '#app/lib/statusCodes.const';
+
+const retrieveDownloads = async (service: String) => {
```

Comment:
Could use the global `Services` type here:
```suggestion
const retrieveDownloads = async (service: Services) => {
```